README.md

```diff
@@ -125,7 +125,6 @@ WARNING: You have not specified enough hosts to satisfy the blueprint.
 ```
 $ yaas cluster create cluster1 --template=template1.json
 $ cat template2.json | yaas cluster create cluster2
-$ yaas cluster create cluster3
```

Comment:
I'm not convinced a template is required. It could be a valid workflow to create an empty, "shell" cluster and add hosts/services 1 by 1. If that's a supported workflow, I think we should support it too.